src/pool/srv_pool.c

```diff
@@ -1489,6 +1526,9 @@ handle_event(struct pool_svc *svc, struct pool_svc_event_set *event_set)
 
 		if (event->psv_src != CRT_EVS_SWIM || event->psv_type != CRT_EVT_ALIVE)
 			continue;
+
+		pool_restart_rebuild_if_rank_wip(svc->ps_pool, event->psv_rank);
```

Comment:
[Question] As @liw mentioned last week, if this ALIVE event means rank state from SUSPECT to ALIVE , and there is a rebuild running, and its rebuild status is not finished, we will abort rebuild unexpected ?

Reply:
I don't know what method can differenciate/avoid that case, even one engine crashed and reboot again in short time, it possibly reflect as SUSPECT+ALIVE.
Seems in SWIM protocol, it is not required to report ALIVE event when remove from SUSPECT status.

Reply:
as we discussed in team meeting, if the ALIVE event prove to be a problem later, we can refine some handling in cart/swim. currently can leave it as now.